Rakefile

```diff
@@ -170,6 +179,17 @@ namespace "ebpf" do
     sh "#{sudo} docker build -t #{DOCKER_IMAGE} -f #{DOCKER_FILE} ."
   end
 
+  desc "Format C code using clang-format"
+  task :cfmt do
+    files = `find ebpf/c -name "*.c" -o -name "*.h"`.split("\n")
+    files.each do |file|
+      sh "clang-format -i -style='{BasedOnStyle: WebKit, BreakBeforeBraces: Attach}' #{file}"
+      # This only works with gnu sed
+      sh "sed -i 's/__attribute__((always_inline)) /__attribute__((always_inline))\\
```

Comment:
Is there a clang format rule we can use for this?  The GNU-only sed thing is unfortunate, but I guess acceptable since everyone devving on the C code will need to do so in a VM

Reply:
I did not found a format rule for this :/ a workaround would be to wrap the macro around clang ignore comments.

To make it work on macos it just needs a `brew install gnu-sed --with-default-names`